tests/snapshots/TestCLICommands_atmos_describe_config.stdout.golden

```diff
@@ -47,12 +47,6 @@
   "integrations": {
     "atlantis": {}
   },
-  "schemas": {
-    "jsonschema": {},
-    "cue": {},
-    "opa": {},
-    "atmos": {}
-  },
```

Comment:
Is this deliberate? Are we suppressing them because they are empty?

Reply:
Yes, that is true, we are suppressing them because they are empty. But this is not entirely deliberate as well. We are making schema dynamic to acomodate the user defined schemas so we converted to a map[string]interface{} instead of the custom schema struct we had. This also lead to the collapse of the empty keys.